ios/SecuritySdk/Classes/Digital Signature/DigitalSignature.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DigitalSignature.swift
+//  SecuritySdk
+//
+
+import Foundation
+
+/// Digital Signature is an external wrapper that provides both the
+///  signing curve algorithm used for signing and the signature data
+public struct DigitalSignature {
```

Comment:
why are we coupling these two things here (what does it buy us), as opposed to, e.g. raw signature with accompanying cert?

Reply:
Its a wrapper to the higher abstraction to provide context. If we were to propagate the signature bytes, we should also provide which signing algorithm was used. For example when generating a signed CSR, the signer is of a higher abstraction that does not distinguish the underlying signer type (i.e the signing algorithm used).